### DISCUSSION TITLE
How do you update Paru?

### DISCUSSION THREAD

**Discussion by Dosx001** [Morganamilo/paru#612] [General]
When I run ```paru -Syu```, I get the following message.
```
:: marked out of date: paru
```
I'm a new Arch user and recently installed Paru.
```
$ paru -V
paru v1.9.0 - libalpm v13.0.1
```
On GitHub I see the current release is v1.9.1.

Is there an easy way to update or do I have to manually reinstall?

**Comment by Morganamilo** [comment]
The aur package is out of date. I plan to skip the 1.9.1 release and go straight to 1.9.2. Only reasy 1.9.1 got published is cause it was already tagged and for consistency it made sense.
